src/examples/draggable/DropzoneDanger.tsx

```diff
@@ -0,0 +1,254 @@
+        isPlaceholder={isDragging}
+        id={id}
+        value={value}
+        ref={setActivatorNodeRef}
+      />
+    </DraggableList.Item>
+  );
+};
+
+const DroppableColumn = ({ isActive, isHighlighted }: IDroppableColumnProps) => {
+  const { setNodeRef } = useDroppable({ id: 'dropzone' });
+  const message = isHighlighted ? 'Drop to delete' : 'Drag fruits here';
+
+  return (
+    <StyledDropzone isDanger ref={setNodeRef} isActive={isActive} isHighlighted={isHighlighted}>
+      <Dropzone.Message>{message}</Dropzone.Message>
+    </StyledDropzone>
+  );
+};
+
+const Example = () => {
+  const [items, setItems] = useState<IColumnItem[]>(defaultItems);
+  const [activeId, setActiveId] = useState<string | null>(null);
+  const [dropzoneIsHighlighted, setDropzoneIsHighlighted] = useState<boolean>(false);
+
+  const theme = useTheme();
+  const overlayRef = useRef<HTMLDivElement>(null);
+
+  const activeItem = items.find(item => item.id === activeId);
+
+  const sensors = useSensors(
+    useSensor(MouseSensor),
+    useSensor(TouchSensor),
+    useSensor(KeyboardSensor, { coordinateGetter })
+  );
+
+  const onDragStart = useCallback(({ active }: DragStartEvent) => {
+    setActiveId(active.id as string);
+  }, []);
+
+  const onDragOver = useCallback(({ over }: DragOverEvent) => {
+    setDropzoneIsHighlighted((over?.id as string) === 'dropzone');
+  }, []);
+
+  const onDragEnd = useCallback(
+    ({ active, over }: DragEndEvent) => {
+      const overId = over?.id;
+
+      if (overId === 'dropzone') {
+        setItems(items.filter(item => item.id !== active.id));
+      }
+
+      setActiveId(null);
+      setDropzoneIsHighlighted(false);
+    },
+    [items]
+  );
+
+  const onDragCancel = useCallback(() => {
+    setActiveId(null);
+    setDropzoneIsHighlighted(false);
+  }, []);
+
+  return (
+    <DndContext
+      accessibility={{ announcements }}
+      sensors={sensors}
+      onDragStart={onDragStart}
+      onDragOver={onDragOver}
+      onDragEnd={onDragEnd}
+      onDragCancel={onDragCancel}
+      measuring={{ droppable: { strategy: MeasuringStrategy.Always } }}
+    >
+      <Row justifyContent="center">
+        <Col size={4}>
+          <StyledHeading isBold>List of produce</StyledHeading>
+          <DraggableList>
+            {items.map(item => (
+              <DraggableListItem key={item.id} {...item} />
+            ))}
+          </DraggableList>
+        </Col>
+        <Col size={5}>
+          <StyledHeading isBold>Yucky fruits</StyledHeading>
+          <DroppableColumn isActive={!!activeId} isHighlighted={dropzoneIsHighlighted} />
+        </Col>
+        <DragOverlay>
+          {activeItem && (
+            <div style={{ padding: `${theme.space.xxs} 0` }}>
+              <DraggableItem
+                ref={overlayRef}
+                id={activeItem.id}
+                value={activeItem.value}
+                isOverlay
+                isGrabbed
+              />
+            </div>
```

Comment:
Would this be better as a `Styled` component vs. the need to setup the theming context?